### DISCUSSION TITLE
I have been trying to integrate my jellyfin with Reiverr

### DISCUSSION THREAD

**Discussion by karthiceaswar** [aleksilassila/reiverr#80] [Q&A] [answered]
Great app by the way. So I installed reiverr and i loved it. I have been trying to integrate my jellyfin server with it. Created an API key from jellyfin ui: settings->dashboard->api keys->new->name: Reiverr. It gives me an API key. My all docker containers are all under a custom docker network. So when I paste this API key and even when i used my public IP to access it, its saying couldn’t connect. I know there's got to be something i am missing but no idea. Pls help me to configure it with jellyfin and eventually i will also integrate sonarr and radarr. Thank you.
<img width="762" alt="Screenshot 2023-08-26 at 3 46 42 PM" src="https://github.com/aleksilassila/reiverr/assets/74752653/09a04d04-6464-4ddc-be98-a1da13ce5f89">
<img width="1421" alt="Screenshot 2023-08-26 at 3 44 28 PM" src="https://github.com/aleksilassila/reiverr/assets/74752653/0c54163a-9472-4b60-ba5e-3e436d3094bc">
<img width="425" alt="Screenshot 2023-08-26 at 3 45 02 PM" src="https://github.com/aleksilassila/reiverr/assets/74752653/cfa5264c-cf66-44fe-a238-9dcb8cb753c8">
<img width="382" alt="Screenshot 2023-08-26 at 3 45 44 PM" src="https://github.com/aleksilassila/reiverr/assets/74752653/3e31ab1f-6488-489e-a189-c8fdd2498d2f">
p.s I revoked the API key

**Comment by aleksilassila** [comment]
Is the url the same that you use to access jellyfin web ui? If yes, could you check your logs and see if there are any errors, if yes screenshot the requests that are failing

**Reply by karthiceaswar** [reply, marked as answer]
Got it to work, mistake from my side. Added '/' at the end of the url, removed it and it worked.

**Reply by karthiceaswar** [reply]
One more small query, now i am using my public Ip/domain name to connect, but is it possible to access it with in the vps docker network itself ? tried http://127.0.0.1:8096 with no luck. All these container are connected to the same docker network. Since it is a cloud linux vps i have no way of verifying it, but wget http://127.0.0.1:8096 gives me index.html with title jellyfin

**Reply by aleksilassila** [reply]
Currently, since the requests are made from the browser, there’s no other way unfortunately. If you can’t use the URL to access jellyfin web, it won’t work in Reiverr either
